docs/markdown/intro/README.md

```diff
@@ -236,23 +238,41 @@ You can also import component groups and directives as Vue plugins by importing
 or directive directory:
 
 ```js
-// This imports all the layout components such as <b-container>, <b-row>, <b-col>:
+// This imports all the layout components such as <b-container>, <b-row>, <b-col>
 import { Layout } from 'bootstrap-vue/es/components'
 Vue.use(Layout)
 
-// This imports <b-modal> as well as the v-b-modal directive as a plugin:
+// This imports <b-modal> as well as the v-b-modal directive as a plugin
 import { Modal } from 'bootstrap-vue/es/components'
 Vue.use(Modal)
 
-// This imports <b-card> along with all the <b-card-*> sub-components as a plugin:
+// This imports <b-card> along with all the <b-card-*> sub-components as a plugin
 import { Card } from 'bootstrap-vue/es/components'
 Vue.use(Card)
 
-// This imports directive v-b-scrollspy as a plugin:
+// This imports directive v-b-scrollspy as a plugin
 import { Scrollspy } from 'bootstrap-vue/es/directives'
 Vue.use(Scrollspy)
 ```
 
+Alternatively, you can import Plugins as follows:
```

Comment:
There is no reason to introduce such alternative IMO. This breaks tree shake!

Reply:
Would destructuring BootstrapVue into componentPlugins and directivePlugins constants help?

Reply:
We could add named exports to main `index.js`, but that would probably break when using esm build (making people need to use `module.default`)?

Reply:
@tmorehouse  Yes. destructuring + named exports should work but would be better to first test such usage in next RC before adding to the docs as an official way.

Reply:
OK... we can revert the changes for the doc additions.

Reply:
i.e. 
```js
import BootstrapVue from 'bootstrap-vue'
const { componentPlugins: { Layout, Modal } } = BoostrapVue

Vue.use(Layout)
Vue.use(Modal)
```